docs/howto/openstack/nova/boot-image-volume.md

```diff
@@ -0,0 +1,395 @@
+    |            | balancer_member,_member_,swiftoperator,creator', hw_machine_type='pc',              |
+    |            | hw_qemu_guest_agent='yes', image_type='snapshot',                                   |
+    |            | instance_uuid='19ffeb4d-3016-481f-886e-6caecc525fc2', locations='[]',               |
+    |            | os_hidden='False', owner_project_name='Meanwhile-in-Drama',                         |
+    |            | owner_user_name='kolderson', user_id='c096cf99f65a4d22a6954b67d2ec11d7'             |
+    | protected  | False                                                                               |
+    | schema     | /v2/schemas/image                                                                   |
+    | status     | queued                                                                              |
+    | tags       |                                                                                     |
+    | updated_at | 2023-05-10T13:31:46Z                                                                |
+    | visibility | private                                                                             |
+    +------------+-------------------------------------------------------------------------------------+
+    ```
+
+    At first, the status of the snapshot is `queued`. To make sure the
+    snapshot has been successfully created, type:
+
+    ```bash
+    openstack image show nanisivik_snap -c status -f value
+    ```
+
+    You should see this:
+
+    ```plain
+    active
+    ```
+
+## Creating a volume from the snapshot
+
+=== "{{gui}}"
+    After a few seconds, the snapshot will be created and listed in the
+    *Snapshots* tab. Before you proceed, take note of its size. At the
+    right-hand side of the snapshot row, click the :fontawesome-database:
+    icon to create a new volume off of the snapshot you just created.
+
+    ![Creating volume from snapshot](assets/bfi-to-bfv/shot04.png)
+
+    A new vertical pane will slide over from the right-hand side of the
+    {{gui}}. Type in a name for the new volume, and choose a volume size
+    **bigger** than the snapshot size. Then, you need to type in a
+    description regarding the new volume.
+
+    ![Typing in name, size, and description](assets/bfi-to-bfv/shot05.png)
+
+    Scroll down a bit if you have to, and click the green *Create* button.
+
+    ![Starting volume creation](assets/bfi-to-bfv/shot06.png)
+=== "OpenStack CLI"
+    Before creating a volume off of the snapshot you just created, jot down
+    its size like this:
+
+    ```bash
+    openstack image show nanisivik_snap -c min_disk -f value
+    ```
+
+    In our example, the size of the snapshot is `50`
+    [gibibytes](https://en.wikipedia.org/wiki/Gigabyte#Base_2_(binary)).
+    Now, go ahead and create a volume slightly larger than the size of the
+    snapshot:
+
+    ```bash
+    openstack volume create --size 56 --image nanisivik_snap nanisivik_vol
```

Comment:
Any particular reason for bumping the size by more than 10%?

Reply:
I just like multiples of 8 :) Ideally, 50GiB should not exist. Acceptable values in that vicinity would be 48GiB or 56GiB. But the world is a strange place, so...